demo/JD-recommendation/README.md

```diff
@@ -0,0 +1,62 @@
+For full instructions, please refer to https://github.com/4paradigm/OpenMLDB/blob/main/docs/zh/use_case/JD_recommendation.md 
+
+#Training:
+1. Engage openmldb for feature extraction:
+##in openmldb docker
+docker exec -it demo bash
```

Comment:
how to download the `demo` image

Reply:
readme is a bit old, I'll rewrite it in next pr.